.circleci/publish.sh

```diff
@@ -1,34 +0,0 @@
-#!/bin/bash
-set -euo pipefail
```

Comment:
I think this file is still necessary. It wasn't actually used by circle ci. It was using by the publish github action.